lib/full-site-editing/template-parts.php

```diff
@@ -168,19 +168,36 @@ function set_unique_slug_on_create_template_part( $post_id ) {
  * @return array The supported template part area values.
  */
 function gutenberg_get_allowed_template_part_areas() {
-	$default_area_values = array(
-		WP_TEMPLATE_PART_AREA_HEADER,
-		WP_TEMPLATE_PART_AREA_FOOTER,
-		WP_TEMPLATE_PART_AREA_SIDEBAR,
-		WP_TEMPLATE_PART_AREA_UNCATEGORIZED,
+	$default_area_definitions = array(
+		array(
+			'area'        => WP_TEMPLATE_PART_AREA_UNCATEGORIZED,
+			'label'       => __( 'General' ),
```

Comment:
I believe we should we add the `gutenberg` domain to `__()` calls.

Reply:
ah right! thanks.  updated 😁